### ISSUE TITLE
Local development support

### ISSUE THREAD

**Issue by DJM0** [lelouch77/docusaurus-lunr-search#27]
From the README.md

> Note: Docusaurus search information can only be generated from a production build. Local development is currently not supported.

It would be nice to have this working in the dev server, Maybe a niche use case but I use Docusaurus as a way to render local notes, and I only ever use the development server (live reloading etc) and never generate production builds.

I know I'm likely missing something as to why this isn't possible. In the meantime if this isn't a planned feature it might be nice to hide or visually disable the search box while in dev mode.

**Comment by ahmad-kassem** [comment]
> From the README.md
> 
> > Note: Docusaurus search information can only be generated from a production build. Local development is currently not supported.
> 
> It would be nice to have this working in the dev server, Maybe a niche use case but I use Docusaurus as a way to render local notes, and I only ever use the development server (live reloading etc) and never generate production builds.
> 
> I know I'm likely missing something as to why this isn't possible. In the meantime if this isn't a planned feature it might be nice to hide or visually disable the search box while in dev mode.

Yeah it would be nice to have a workaround for this, or if just we can disable the search when we are running server in dev mode.
